uniffi_bindgen/src/scaffolding/templates/namespace_metadata.rs

```diff
@@ -0,0 +1,12 @@
+
+/// Export namespace metadata.
+///
+/// See `uniffi_bindgen::macro_metadata` for how this is used.
+{%- let const_var = "UNIFFI_META_CONST_NAMESPACE_{}"|format(ci.namespace().to_shouty_snake_case()) %}
+{%- let static_var = "UNIFFI_META_NAMESPACE_{}"|format(ci.namespace().to_shouty_snake_case()) %}
+
+const {{ const_var }}: ::uniffi::MetadataBuffer = ::uniffi::MetadataBuffer::from_code(::uniffi::metadata::codes::NAMESPACE)
+    .concat_str(module_path!()) // This is the crate name, since we're in the crate root
```

Comment:
Could use `env!("CARGO_CRATE_NAME")` here. ([docs](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates))